yugabyte/yugabyte-db
# [Docs] Track UTMs to monitor traffic driving sign-ups for YBM

Marketing Team is on process to Track UTMs that drive traffic to the docs domain with the goal to track conversions/sign ups for YBM. This PR will create cookies for the UTM parameters and pass to the YBM signup.

**Testing Scenario:**
1. Open https://deploy-preview-19672--infallible-bardeen-164bc9.netlify.app/?utm_source=google&utm_medium=paid_search&utm_campaign=docs
2. Navigate to any page and then click on the YBM Signup link (You can easily find it in header under "Start Now" button)
3. YBM Signup should looks like this: https://cloud.yugabyte.com/signup?utm_source=google&utm_medium=paid_search&utm_campaign=docs
